### ISSUE TITLE
Feature Request/Idea

### ISSUE THREAD

**Issue by Timongcraft** [PetyXbron/minecraft-bot#13]
May you can make a language system were there is a folder like `lang` and there is usualy an `en.json` but you can add your own and can change it in the config.json

(If this will bet build-in I would make german)

**Comment by PetyXbron** [comment]
**Why @Timong-craft?**
You can translate all editable messages/text/responses in [config.yml](https://github.com/PetyXbron/minecraft-bot/blob/main/config.js) [at line 86 and down](https://github.com/PetyXbron/minecraft-bot/blob/main/config.js#L86-L182).

**Comment by Timong-craft** [comment]
Yes But I don't know if I should also translate the word 'command' becorse in german it is 'Befehl' but thats awkward

**Comment by PetyXbron** [comment]
I am thinking about trying to add something like this in some of the new updates.
